docs/update_and_migration/migrate_to_ibexa_dxp/migrating_from_ez_publish_platform.md

```diff
@@ -242,6 +242,69 @@ Import to your database the changes provided in one of the following files. It's
 
 Steps here should only be done once you are ready to move away from legacy and Legacy Bridge, as the following Field Types are not supported by legacy. In other words, content you have migrated will not be editable in legacy admin interface anymore, but rather in the more modern eZ Platform back-end UI, allowing you to take full advantage of what the Platform has to offer.
 
+The Field Types unsupported in eZ Platform are:
+
+- [XmlText](#321-migrate-xmltext-content-to-richtext)
+- [Page](#322-migrate-page-field-to-page-ez-enterprise-only)
+- Star Rating
+
+For Field Types which do not have specific procedures below, you must either:
+
+- implement them yourself in eZ Platform, or
+- remove them from all Content Types that use them.
+
+!!! tip
+
+    To find out which Content Types use a specific Field Type,
+    you can run the following SQL query on your database (in this case, for the Star Rating Field Type):
+
+    ``` sql
+    select contentclass_id from ezcontentclass_attribute where data_type_string='ezsrrating' group by contentclass_id;
+    ```
+
+??? tip "`defaultLayout` setting not available"
```

Comment:
About this section.. This section applies if you upgrade to 2.5 and then realizes that you still have old page fields that hasn't been converted and you want to delete them. IE: you want to delete the fields using the admin-ui in 2.5. This also applies if you are intending to run 2.5 with legacy bridge. Then the config below should to be adapted to the config you have in legacy ( so adding any custom layouts  etc. ).

If you realize you have page fields you want to delete prior to migration, you may just do that in the legacy admin without applying the config below.
Sorry that I was not clear in this in our earlier chat

If you

Reply:
Does this also refer to upgrade to 1.13? Or do you only do this on 1.13 > 2.5?

Reply:
It applies to the moment you drop legacy support ( ie move for ezpublish platform to platform, or get rid of legacy bridge in ezplatform )